seq2science/scripts/deseq2/deseq2.R

```diff
@@ -95,19 +100,55 @@ if (n_DEGs == 0) {
 ## generate additional files if DE genes are found
 
 # generate MA plot (log fold change vs mean gene counts)
-output_ma_plot <- sub(".diffexp.tsv", ".ma_plot.pdf", output)
 b <- ifelse(is.na(batch), '', ', batch corrected')
 title <- paste0(
   groups[1], ' vs ', groups[2], '\n',
   n_DEGs, ' of ', nrow(reduced_counts), ' DE (a = ', fdr, b, ')'
 )
-pdf(output_ma_plot)
+
+png(output_mavol_plot, width = 465, height = 225, units='mm', res = 300)
+layout(1:2)
+#par(mfrow=c(1,2))
```

Comment:
@siebrenf any clue why this doesn't work? When I plotMA twice, it works (at least tested with par()), in the sense I get two plots next to each other. But every time I try to plot with EnhancedVolcano I just get one plot... :sob:

Reply:
might be the package can't work with `par()` natively? Here's a potentially useful [BS post](https://www.biostars.org/p/401067/) (gotta love the abbreviation)

Reply:
Yeah so enhancedVolcano works with grid and par. But not deseq2::maplot....

Reply:
yeah so in that example they use gridspec, but deseq2::maplot doesnt work with that.... :shrug:

Reply:
![image](https://user-images.githubusercontent.com/48289046/135643666-19666b5c-2fcd-4aa6-b45c-07d1e6f9d8db.png)